integration/feature/init/src/MillInitMavenTests.scala

```diff
@@ -0,0 +1,221 @@
+        // cannot resolve native dependencies defined by build extension os-maven-plugin
+        initRes.out.contains(
+          """[codec-http2] dropping classifier ${os.detected.classifier} for dependency io.netty:netty-tcnative:2.0.66.Final
+            |[transport-native-epoll] dropping classifier ${os.detected.classifier} for dependency io.netty:netty-tcnative:2.0.66.Final
+            |[transport-native-kqueue] dropping classifier ${os.detected.classifier} for dependency io.netty:netty-tcnative:2.0.66.Final
+            |[handler] dropping classifier ${os.detected.classifier} for dependency io.netty:netty-tcnative:2.0.66.Final
+            |[example] dropping classifier ${os.detected.classifier} for dependency io.netty:netty-tcnative:2.0.66.Final
+            |[testsuite] dropping classifier ${os.detected.classifier} for dependency io.netty:netty-tcnative:2.0.66.Final
+            |[testsuite-shading] dropping classifier ${os.detected.classifier} for dependency io.netty:netty-tcnative:2.0.66.Final
+            |[transport-blockhound-tests] dropping classifier ${os.detected.classifier} for dependency io.netty:netty-tcnative:2.0.66.Final
+            |[microbench] dropping classifier ${os.detected.classifier} for dependency io.netty:netty-tcnative:2.0.66.Final
+            |""".stripMargin
+        ) ==> true
+        initRes.out.contains(
+          "generated 47 Mill build file(s)"
+        ) ==> true
+        initRes.isSuccess ==> true
+
+        val resolveRes = eval(("resolve", "_"))
+        Seq(
+          "all",
+          "dev-tools",
+          "common",
+          "buffer",
+          "codec",
+          "codec-dns",
+          "codec-haproxy",
+          "codec-http",
+          "codec-http2",
+          "codec-memcache",
+          "codec-mqtt",
+          "codec-redis",
+          "codec-smtp",
+          "codec-socks",
+          "codec-stomp",
+          "codec-xml",
+          "resolver",
+          "resolver-dns",
+          "resolver-dns-classes-macos",
+          "resolver-dns-native-macos",
+          "transport",
+          "transport-native-unix-common-tests",
+          "transport-native-unix-common",
+          "transport-classes-epoll",
+          "transport-native-epoll",
+          "transport-classes-kqueue",
+          "transport-native-kqueue",
+          "transport-rxtx",
+          "transport-sctp",
+          "transport-udt",
+          "handler",
+          "handler-proxy",
+          "handler-ssl-ocsp",
+          "example",
+          "testsuite",
+          "testsuite-autobahn",
+          "testsuite-http2",
+          "testsuite-osgi",
+          "testsuite-shading",
+          "testsuite-native",
+          "testsuite-native-image",
+          "testsuite-native-image-client",
+          "testsuite-native-image-client-runtime-init",
+          "transport-blockhound-tests",
+          "microbench",
+          "bom"
+        ).forall(resolveRes.out.contains) ==> true
```

Comment:
This kind of long-multi-line-expression followed by a single `.forall` is kind of hard to read; let's instead assign the `Seq` to a local `val` and then call `.forall` on that `val` after